docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
       retries: 3
       start_period: 30s
     environment:
+      # Run "composer require symfony/orm-pack" to install and configure Doctrine ORM
+      DATABASE_URL: postgresql://${POSTGRES_USER:-symfony}:${POSTGRES_PASSWORD:-ChangeMe}@db:5432/${POSTGRES_DB:-app}?serverVersion=${POSTGRES_VERSION:-13}
```

Comment:
Maybe add an additional commented out line with mysql/mariadb configuration? I see that question way too often on the Symfony Devs Slack channel.

Reply:
There is already a message explaining how to configure MySQL/Maria in the recipe's post install message.

For Docker, a comment will not be enough because the Dockerfile and `docket-compose.yml` must also be updated.